### PR TITLE
Use binary log format for TriFinger and add script for converting log

### PR DESCRIPTION
**Pull request by luator**
## Description

- Use the binary log format in trifinger_backend.py
- Add a script to convert binary log file to csv file.

## How I Tested

See https://github.com/open-dynamic-robot-initiative/robot_interfaces/pull/81.

## Do not merge before

[//]: # "Link other open pull request here that need to be merged first."
[//]: # "Remove this section if it does not apply."

- [x] https://github.com/open-dynamic-robot-initiative/robot_interfaces/pull/81

## I fulfilled the following requirements

[//]: # "Please make sure you followed these steps before requesting a review."
[//]: # "Check the boxes in the list below, when done."

- [x] All new code is formatted according to our style guide (for C++ run clang-format, for Python, run flake8 and fix all warnings).
- [x] All new functions/classes are documented and existing documentation is updated according to changes.
- [x] No commented code from testing/debugging is kept (unless there is a good reason to keep it).

### CODE REVIEW

File: scripts/robot_log_dat2csv.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        help="Number of fingers of the robot.",
+    )
+    args = parser.parse_args()
+
+    if args.number_of_fingers == 1:
+        module = robot_interfaces.finger
+    else:
+        module = robot_interfaces.trifinger
+
+    log = module.LogReader(args.infile)
```
Comment (shrutij01) [inline scripts/robot_log_dat2csv.py:28]: so like BinaryLogReader (referring to my comment on the other PR) here to be more explicit, then it's clear what's happening just as the method in `trifinger_backend` is `write_current_buffer_binary`

**Review by shrutij01 — APPROVED** [review]
looks good to me :)

File: scripts/robot_log_dat2csv.py (hunk 1 above)
Comment (luator) [reply scripts/robot_log_dat2csv.py]: done

**Review by shrutij01 — APPROVED** [review]
looks good!

**Pull request merged**